### PR TITLE
[#347] As a developer, I can have a lane in Fastfile.swift for clean up data

### PR DESCRIPTION
**Pull request by nkhanh44**
#347 

## What happened

- Created utility functions to clean up xcov output and derived data.
 
## Insight

- `cleanUpDerivedData` function for clean up derived data at path `./DerivedData`.
- `cleanUpXcov` function for cleaning up output of xcov at path `./fastlane/xcov_output`.
 
## Proof Of Work
Screen Shot 2022-09-30 at 10 44 07

### CODE REVIEW

File: fastlane/Constants/Constant.swift [hunk 1]
```diff
@@ -33,6 +33,7 @@ enum Constant {
     static let outputPath = "./Output"
     static let buildPath = "\(outputPath)/Build"
     static let derivedDataPath = "\(outputPath)/DerivedData"
+    static let xcovOutputDirectoryPath: String = "./fastlane/xcov_output"
```
Comment (vnntsu) [inline fastlane/Constants/Constant.swift:36]: Could we move `xcov_output` to `Output`?

File: fastlane/Fastfile.swift [hunk 2]
```diff
@@ -68,4 +68,16 @@ class Fastfile: LaneFile {
         Symbol.downloadFromAppStore(versionNumber: versionNumber, buildNumber: buildNumber)
         Symbol.uploadAppStoreToCrashlytics(versionNumber: versionNumber, buildNumber: buildNumber)
     }
+
+    // MARK: - Utilities
+
+    func cleanUpDerivedData() {
```
Comment (vnntsu) [inline fastlane/Fastfile.swift:74]: I think we can have only one clean function to clear all things in `Output` folder.

File: fastlane/Constants/Constant.swift (hunk 1 above)
Comment (nkhanh44) [reply fastlane/Constants/Constant.swift]: Yeah, I update at https://github.com/nimblehq/ios-templates/pull/366/commits/e8eef877f1d68b1bc3f281aa0b3077a7d199178e

File: fastlane/Fastfile.swift (hunk 2 above)
Comment (nkhanh44) [reply fastlane/Fastfile.swift]: Correct! I updated at https://github.com/nimblehq/ios-templates/pull/366/commits/e8eef877f1d68b1bc3f281aa0b3077a7d199178e

**Review by suho — APPROVED** [review]
LGTM

**Review by vnntsu — APPROVED** [review]
LGTM! 🚀

**Review by ducbm051291 — APPROVED** [review]
LGTM

**Pull request merged**